attest/verifier/types/src/verification.rs

```diff
@@ -151,6 +154,63 @@ impl Message for VerificationSignature {
     }
 }
 
+/// Structure for holding the contents of the Enclave's Report Data.
+/// The Enclave's ReportData member contains a SHA256 hash of this structure's
+/// contents.
+#[derive(Debug, Clone)]
+pub struct EnclaveReportDataContents {
```

Comment:
I'm up for suggestions on this name.  
I went a little verbose because `ReportData` is a `mc-sgx-core-type` that is specific to the `Report.report_data` field.  I'm guessing the field was named early on in Intel's sgx development as they seem to store hashes or signatures in this field as well. So it's more of a digest than the actual data. This struct here is meant to the actual data.

I'm generally not fond of "metadata"

Maybe `ReportDataForReals`

Reply:
I think this name is fine